transformer/dockerfilegenerator/dotnet5dockerfiletransformer.go

```diff
@@ -120,7 +122,7 @@ func (t *DotNet5DockerfileGenerator) DirectoryDetect(dir string) (services map[s
 			logrus.Errorf("Could not parse the project file %s", err)
 		}
 		serviceName := strings.TrimSuffix(filepath.Base(csprojFile), filepath.Ext(csprojFile))
-		if configuration.PropertyGroup.TargetFramework == dotNETCore5 {
+		if configuration.PropertyGroup.TargetFramework == dotNETCore5 || configuration.PropertyGroup.TargetFramework == netcoreapp || configuration.PropertyGroup.TargetFramework == netstandard {
```

Comment:
If everything (code structure, config to fetch ..) is same across all three versions, just declare an array and use common.IsStringPresent function.

Reply:
Done.